control-plane/plugin/src/resources/node.rs

```diff
@@ -142,17 +135,36 @@ impl Cordoning for Node {
                 }
                 OutputFormat::None => {
                     // In case the output format is not specified, show a success message.
-                    let cordon_labels = node
-                        .into_body()
-                        .spec
-                        .map(|node_spec| node_spec.cordon_labels)
-                        .unwrap_or_default();
-                    if cordon_labels.is_empty() {
+                    let mut cordon_labels: Vec<String> = vec![];
+                    let mut drain_labels: Vec<String> = vec![];
+                    match node.clone().into_body().spec.unwrap().cordondrainstate {
+                        Some(ds) => {
+                            cordon_labels = match ds {
+                                CordonDrainState::cordonedstate(state) => state.cordonlabels,
+                                CordonDrainState::drainingstate(state) => state.cordonlabels,
+                                CordonDrainState::drainedstate(state) => state.cordonlabels,
```

Comment:
Can we move this to a function to which takes a `CordonDrainState` and gives the cordon labels, similarly for drain labels. Seems to be repeated at quite a few places.